include/mata/nfa.hh

```diff
@@ -1354,6 +1510,26 @@ public:
         return alphabet;
     }
 
+    /**
+     * Create alphabet from vector of of NFAs.
+     * @param[in] nfas Vector of pointers to NFAs to create alphabet from.
+     * @return Created alphabet.
+     */
+    static EnumAlphabet from_nfas(const AutPtrSequence& nfas) {
+        EnumAlphabet alphabet{};
+        size_t nfa_num_of_states{};
+        for (const Nfa* const nfa: nfas) {
```

Comment:
I know there is probably no generic way, but this and the previous functions are so similar that it screams for refactoring.

Reply:
Anything connected to alphabets is temporary until we figure out how to implement alphabets into `Nfa` struct and how to handle special characters etc.

Nonetheless, you are of course right about this. I will try to refactor these functions.

Reply:
Done.